build.gradle

```diff
@@ -1,47 +1,38 @@
-/*
- * This build file was generated by the Gradle 'init' task.
- *
- * This generated file contains a sample Java project to get you started.
- * For more details take a look at the Java Quickstart chapter in the Gradle
- * user guide available at https://docs.gradle.org/4.3/userguide/tutorial_java_projects.html
- */
-
-// Apply the java plugin to add support for Java
 apply plugin: 'java'
-
-// Apply the application plugin to add support for building an application
 apply plugin: 'application'
 
+mainClassName = 'HelloJavaMesh'
 applicationDefaultJvmArgs = ["-noverify"]
+
 buildscript {
     ext {
         artifactory_app_username = "insertyourusername"
         artifactory_app_password = "insertyourpasswordhash"
         artifactory_app_key = "insertyourkey"
     }
     repositories {
+        jcenter()
+        mavenCentral()
         maven {
-            url "http://research.rightmesh.io/artifactory/libs-local"
+            url "https://research.rightmesh.io/artifactory/libs-local"
             credentials {
                 username artifactory_app_username
                 password artifactory_app_password
             }
         }
-        mavenCentral()
-        jcenter()
     }
     dependencies {
-        classpath 'io.left.rightmesh:rightmesh-plugin:1.3'
+        classpath 'io.left.rightmesh:rightmesh-plugin:1.6'
     }
 }
 apply plugin: 'io.left.rightmesh.rightmesh-plugin'
 build.dependsOn("rightmesh")
 
 repositories {
-    mavenCentral()
     jcenter()
+    mavenCentral()
     maven {
-        url "http://research.rightmesh.io/artifactory/libs-local"
+        url "https://research.rightmesh.io/artifactory/libs-local"
```

Comment:
This is how you can tell we haven't updated this code in a while - feels like ages ago we added https